test/integration/helpers/Env.cs

```diff
@@ -23,19 +23,28 @@ private static void Init()
                     var path = AppDomain.CurrentDomain.BaseDirectory;
                     var sr = new StreamReader(path + "env.json");
                     var jsonString = sr.ReadToEnd();
-                    _env = JsonConvert.DeserializeObject<Dictionary<string, string>>(jsonString);
+                    _env = JsonSerializer.Deserialize<Dictionary<string, JsonElement>>(jsonString, new JsonSerializerOptions
+                    {
+                        PropertyNameCaseInsensitive = true
+                    });
                 }
             }
-            catch
+            catch (JsonException jsonEx)
             {
-                _env = new Dictionary<string, string>();
+                Console.WriteLine($"Error parsing JSON: {jsonEx.Message}");
+                _env = [];
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Error initializing environment: {ex.Message}");
+                _env = [];
             }
         }
 
-        private static bool IsTrue(string val)
+        private static bool IsTrue(object val)
         {
-            val = val?.ToLower().Trim();
-            return (val == "true") || (val == "1");
+            val = val?.ToString().ToLower().Trim();
```

Comment:
would this work properly if val is null? In js I'd have something like `val?.ToString()?.ToLower()?.Trim()`

Reply:
since I added the default values for _env, we shouldn't worry of a null case